publisher-html/src/main/resources/templates/events/events.hbs

```diff
@@ -80,6 +80,17 @@
         gap: 4px;
     }
 
+    .accessibility-details {
+        display: flex;
+        background-color: white;
+        color: inherit;
```

Comment:
[nit] Is there a need to inherit the color in this case? (the stroke color seems to default to black)